### PR TITLE
Added a recipe for psr-http-message-bridge

### PR DESCRIPTION
**Pull request by derrabus**
| Q             | A
| ------------- | ---
| License       | MIT
| Doc issue/PR  | N/A

A recipe to integrate symfony/psr-http-message-bridge#89

### CODE REVIEW

File: symfony/psr-http-message-bridge/2.1/config/packages/psr_http_message_bridge.yaml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+services:
+    _defaults:
+        autowire: true
+        autoconfigure: true
+
+    Symfony\Bridge\PsrHttpMessage\HttpFoundationFactoryInterface:
+        class: Symfony\Bridge\PsrHttpMessage\Factory\HttpFoundationFactory
+
+    Symfony\Bridge\PsrHttpMessage\HttpMessageFactoryInterface:
+        class: Symfony\Bridge\PsrHttpMessage\Factory\PsrHttpFactory
```
Comment (nicolas-grekas) [inline symfony/psr-http-message-bridge/2.1/config/packages/psr_http_message_bridge.yaml:10]: it'd be great to find a way to remove those two lines (see my comments on the PR on the bridge)

Comment (derrabus) [reply symfony/psr-http-message-bridge/2.1/config/packages/psr_http_message_bridge.yaml:10]: Okay, let's keep the discussion on the bridge PR.

Comment (nicolas-grekas) [reply symfony/psr-http-message-bridge/2.1/config/packages/psr_http_message_bridge.yaml:10]: You made a good point in the bridge PR: these are useful services on their own.
To really label them as useful services, the interfaces should be *aliases* to the actual implementation. This is the criteria that `debug:autowiring` needs to identify the "useful services".

Comment (derrabus) [reply symfony/psr-http-message-bridge/2.1/config/packages/psr_http_message_bridge.yaml:13]: Both interfaces are aliases now. I've also changed the services in the comment and assigned service IDs to them.

**Review by nicolas-grekas — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Pull request merged**